application/src/main/java/org/togetherjava/tjbot/features/github/PullRequestNotificationRoutine.java

```diff
@@ -0,0 +1,107 @@
+package org.togetherjava.tjbot.features.github;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
+import org.kohsuke.github.GHIssueState;
+import org.kohsuke.github.GHPullRequest;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.GitHubBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.PrNotifications;
+import org.togetherjava.tjbot.db.generated.tables.records.PrNotificationsRecord;
+import org.togetherjava.tjbot.features.Routine;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Routine to send a notification about new pull request.
+ */
+public class PullRequestNotificationRoutine implements Routine {
+
+    private static final Logger logger =
+            LoggerFactory.getLogger(PullRequestNotificationRoutine.class);
+
+    private final Database database;
+    private final String githubApiKey;
+    private Date lastExecution;
+
+    /**
+     * Creates new notification routine.
+     *
+     * @param database the database to get the pull request notifications
+     * @param config the config to get the GitHub API key
+     */
+    public PullRequestNotificationRoutine(Database database, Config config) {
+        this.database = database;
+        this.githubApiKey = config.getGitHubApiKey();
+        this.lastExecution = new Date();
+    }
+
+    @Override
+    public Schedule createSchedule() {
+        return new Schedule(ScheduleMode.FIXED_RATE, 0, 15, TimeUnit.MINUTES);
```

Comment:
Make the time an option in the config or make it a static const. 
Reasons for config is that I'm not sure 15 mins is what we want to set it as just yet. We need to experiment with the timer and have an easy way to update it. 
If we don't make it a config, then a static constant for readability because 15 on it's own is a "magic number".